workshop/main.lua

```diff
@@ -60,8 +60,13 @@ return function(dev)
 
     -- If initialize phase fails, prompt to the error screen
     if (not success) then
+<<<<<<< HEAD
```

Comment:
bad merge here too?

Reply:
Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.